### PR TITLE
Pull request for gslab-python#34: Adds State of Repo Logger

### PR DESCRIPTION
**Pull request by lboxell**

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -52,3 +54,10 @@ if mode == 'release':
     release.release(env, vers, DriveReleaseFiles, local_release, org = 'gslab-econ', repo = 'template')
     ## Specifies default targets to build
     Default('.', local_release)
+
+# Print the state of the repo at end of SCons run
+finish_command = Command( 'state_of_repo.log', [], state_of_repo ) # From http://stackoverflow.com/questions/8901296/how-do-i-run-some-code-after-every-build-in-scons
+Depends(finish_command, BUILD_TARGETS)
+env.AlwaysBuild(finish_command)
+if 'state_of_repo.log' not in BUILD_TARGETS: 
+  BUILD_TARGETS.append('state_of_repo.log')
```
Comment (yuchuan2016) [inline SConstruct]: Check indentation

Comment (yuchuan2016) [inline SConstruct]: Check indentation

**Comment by lboxell** [comment]
@yuchuan2016,

I made the edit and updated the template with the user specified value. Let me know if there is anything else.

**Comment by yuchuan2016** [comment]
@lboxell , I tested by chanfing `maxit=2` and it works as expected. I have no other comments other than [this](https://github.com/gslab-econ/gslab_python/pull/51#issuecomment-275445350).

**Pull request merged**